pkg/ceph/client/image.go

```diff
@@ -41,6 +42,9 @@ func ListImages(context *clusterd.Context, clusterName, poolName string) ([]Ceph
 		return nil, fmt.Errorf("failed to list images for pool %s: %+v", poolName, err)
 	}
 
+	res := regexp.MustCompile(`\[(.*)\]$`).FindStringSubmatch(string(buf))
+	buf = []byte(res[0])
```

Comment:
We should check to make sure there's 1 entry in the slice before indexing into it.  If the slice is empty, should that be an error condition or should we just return an empty slice from this function?

Reply:
I was testing this on a ceph cluster with and without images as well as on INFO and debug log. The results of rbd ls -l are in the playground code. 
I will add the length check anyways an threat it as an empty case since error checking was already done at that line.